dist/preview release/what's new.md

```diff
@@ -1,328 +1,329 @@
-# 5.0.0
```

Comment:
Can you check what is wrong with this file? We cannot compare them :(

Reply:
This is weird. The diff in my IDE looked all good. Will check...

Reply:
Seems my IDE changed line endings to CRLF, no idea why, which made every line change. Fixed this now!